### PR TITLE
increase coverage e2e

### PR DESCRIPTION
**Pull request by drf7**

### CODE REVIEW

File: nuclia_e2e/nuclia_e2e/tests/conftest.py [hunk 1]
```diff
@@ -289,6 +289,22 @@ async def delete_service_account_by_name(
             response.raise_for_status()
         return sa_id
 
+    async def create_vector_set(self, kb_id: str, model: str):
```
Comment (sunbit) [inline nuclia_e2e/nuclia_e2e/tests/conftest.py:292]: this is not on the sdk?

File: nuclia_e2e/nuclia_e2e/tests/test_download_activity_log.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    async with aiohttp.ClientSession() as session, session.get(url) as response:
+        data = []
+        async for line in response.content:
+            data.append(json.loads(line.decode("utf-8")))
+
+        return data
+
+
+@pytest.mark.asyncio_cooperative
+async def test_download_activity_log(regional_api_config: ZoneConfig, email_util: EmailUtil):
```
Comment (sunbit) [inline nuclia_e2e/nuclia_e2e/tests/test_download_activity_log.py:29]: This tests, either:
- put them as part of the kb_features 
- we expect to have chat events, on this kb as other tests use it, so checking the month before is probably safer. now we dont' automatically purge old events, but we'll do eventually (i "detached" some old partitions already)

there's a third option that is doing one request to ensure we have someting, but i thing any of the other two can work. We should try to balance not adding too much new concurrent tests, and trying to reduce (and reuse) expensive operations to a minimum

what do you think?

File: nuclia_e2e/nuclia_e2e/tests/conftest.py (hunk 1 above)
Comment (drf7) [reply nuclia_e2e/nuclia_e2e/tests/conftest.py:292]: No, I didn't find it

File: nuclia_e2e/nuclia_e2e/tests/test_download_activity_log.py (hunk 2 above)
Comment (drf7) [reply nuclia_e2e/nuclia_e2e/tests/test_download_activity_log.py:29]: Okay, I will move the test

Comment (drf7) [reply nuclia_e2e/nuclia_e2e/tests/test_download_activity_log.py:29]: I think I will just get the activity logs from the month before instead of moving it to kb_features, I think it is cleaner if we have the tests separated and not all together if possible

**Review by sunbit — APPROVED** [review]
